data/create_test_db.js

```diff
@@ -1,9 +1,18 @@
 const db = require('./db-config');
 
 db.raw('CREATE DATABASE api-test;')
+
+.then(function(result){
+    console.log('Database `api-test`created\n',result);
+})
+.finally(function(){
+    console.log('Done');
+})
+
```

Comment:
Lines 4-11 here are a duplicate of lines 12-17. To fix this nicely, I suggest running `git checkout origin/main data/create_test_db.js`